### PR TITLE
Restore libusb-no since that Windows 10 only seems to work with that …

### PR DESCRIPTION
**Pull request by JPeterMugaas**
…flag.  Added note about this.

### CODE REVIEW

File: mingw-w64-libgphoto2/PKGBUILD [hunk 1]
```diff
@@ -15,19 +15,20 @@ depends=("${MINGW_PACKAGE_PREFIX}-libsystre"
...
-        ${_realname}-${pkgver}.tar.gz::https://github.com/gphoto/libgphoto2/archive/${_realname}-${pkgver//./_}-release.tar.gz
+makedepends=(
+  "${MINGW_PACKAGE_PREFIX}-gcc"
+  "${MINGW_PACKAGE_PREFIX}-libtool"
+)
+source=(https://sourceforge.net/projects/gphoto/files/libgphoto/${pkgver}/${_realname}-${pkgver}.tar.gz
         libgphoto2-gp_system_filename-fix.patch
-        libgphoto2-use-pkg-config-for-gd.patch) 
-sha256sums=('82118e8abaf54100c6be1d41489f9a26d951c7a6194db6f11efa60274a6f53a7'
+libgphoto2-use-pkg-config-for-gd.patch)
```
Comment (Alexpux) [inline mingw-w64-libgphoto2/PKGBUILD]: Do the same indent as source url. Use spaces, not tabs

File: mingw-w64-libgphoto2/PKGBUILD [hunk 2]
```diff
@@ -15,19 +15,20 @@ depends=("${MINGW_PACKAGE_PREFIX}-libsystre"
          "${MINGW_PACKAGE_PREFIX}-libgd"
          "${MINGW_PACKAGE_PREFIX}-libexif"
          "${MINGW_PACKAGE_PREFIX}-libusb")
-makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
-             "${MINGW_PACKAGE_PREFIX}-libtool")
-source=(#https://sourceforge.net/projects/gphoto/files/libgphoto/${pkgver}/${_realname}-${pkgver}.tar.gz{,.asc}
-        ${_realname}-${pkgver}.tar.gz::https://github.com/gphoto/libgphoto2/archive/${_realname}-${pkgver//./_}-release.tar.gz
+makedepends=(
+  "${MINGW_PACKAGE_PREFIX}-gcc"
+  "${MINGW_PACKAGE_PREFIX}-libtool"
```
Comment (Alexpux) [inline mingw-w64-libgphoto2/PKGBUILD]: Restore indent in makedepends as it was. Please folow or syntax

File: mingw-w64-libgphoto2/PKGBUILD (hunk 1 above)
Comment (Alexpux) [inline mingw-w64-libgphoto2/PKGBUILD]: Fix second comment too

**Comment by peterbud** [comment]
@JPeterMugaas : have you tested this time the functionality of the package? There are a bunch of example / test beds in the libgphoto package.

**Comment by JPeterMugaas** [comment]
Alirhgt, I was able to get gphoto2 to work if you are using libusb without mingw-w64-i686-libusb-usbdk so I think there could be a problem with libusb.  However, although autodetection works, I still get an error about being unable to claim the device.

**Comment by peterbud** [comment]
@JPeterMugaas : you can find the testbeds for libgphoto2 in the [examples folder](https://github.com/gphoto/libgphoto2/tree/master/examples), so there is no immediate need for gphoto2 to work. I have a gphoto2 working properly, but Id like to test more before opening a PR.

**Pull request merged**